src/components/residentPermit/PersonalInfo.tsx

```diff
@@ -94,6 +94,15 @@ const PersonalInfo = ({
       },
     });
   };
+
+  // automatically pre-select the primary address if customer
+  // has at least one address and none selected
+  const isPrimaryAddressSelected =
```

Comment:
I fixed this as part of the reset logic, however it may be solution to https://helsinkisolutionoffice.atlassian.net/jira/software/c/projects/PV/boards/143?modal=detail&selectedIssue=PV-638

Reply:
@danjacob-anders FYI: According to testing, this fix still does not solve this issue completely: https://helsinkisolutionoffice.atlassian.net/browse/PV-638

But it be solved in another PR.